server/sonar-ce/src/main/java/org/sonar/ce/httpd/CeHttpServer.java

```diff
@@ -84,21 +96,49 @@ String getUrl() {
     return "http://" + nanoHttpd.getHostname() + ":" + nanoHttpd.getListeningPort();
   }
 
-  private class SystemInfoNanoHttpd extends NanoHTTPD {
+  private class CeNanoHttpd extends NanoHTTPD {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this a "static" inner class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2694)